### PR TITLE
Update enterprise_logging_apps.adoc

### PR DESCRIPTION
**Pull request by jnweiger**
log level > 1 disagrees with https://github.com/owncloud/admin_audit/blob/master/README.md#configuration
Which claims `All messages regardless of log level will be logged there.`

### CODE REVIEW

**Comment by jnweiger** [comment]
The screenshot is outdated. The admin_audit app is no longer titled `Log user and file sharing actions`. In 2.1.1 ist is named 'Auditing'

image

**Comment by jnweiger** [comment]
This manual entry is a stub. 
Suggest to include: For details see https://github.com/owncloud/admin_audit/blob/master/README.md

**Comment by mmattel** [comment]
@jnweiger @EParzefall 
This is a bigger rewrite because as @jnweiger commented, the current version is less more than a draft. I fixed not only the content and the image, but also sections, naming, missing occ-commands and the splunk integration.

File: modules/admin_manual/pages/configuration/server/occ_commands/app_commands/admin_audit_commands.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+----
+
+[source,console]
+----
+yes
+----
+
+== Delete Ignore CLI Events
+
+This command completely removes the key and the value
```
Comment (EParzefall) [inline modules/admin_manual/pages/configuration/server/occ_commands/app_commands/admin_audit_commands.adoc:37]:
```suggestion
This command completely removes the key and the value:
```

**Review by EParzefall — APPROVED** [review]
Just one minor glitch.

File: modules/admin_manual/pages/configuration/server/occ_commands/app_commands/admin_audit_commands.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
+= Auditing
+
+Marketplace URL: {oc-marketplace-url}/apps/admin_audit[Auditing]
+
+Tracks various activities and actions of your users and admins. For details please see the
```
Comment (jnweiger) [inline modules/admin_manual/pages/configuration/server/occ_commands/app_commands/admin_audit_commands.adoc:5]:
```suggestion
Tracks various activities and actions of your users and admins. For details, please see the
```

Comment (jnweiger) [reply modules/admin_manual/pages/configuration/server/occ_commands/app_commands/admin_audit_commands.adoc:5]: @EParzefall you are the Komma-Expert, I am unsure.

**Comment by jnweiger** [comment]
Wow, looks like a massive improvement. Thank you @mmattel

File: modules/admin_manual/pages/configuration/server/occ_commands/app_commands/admin_audit_commands.adoc (hunk 2 above)
Comment (EParzefall) [reply modules/admin_manual/pages/configuration/server/occ_commands/app_commands/admin_audit_commands.adoc]: @jnweiger  It's an optional comma. I like it.

**Pull request merged**